### ISSUE TITLE
Corneal stromal vascularisation

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10874]
Preferred term label:
Corneal stromal vascularisation

Synonyms
Stromal vascularization of the cornea, Corneal stromal neovascularization

Definition (free text, please give PubMed ID)
The abnormal growth of blood vessels into the corneal stroma, often associated with ocular herpes, chronic inflammation, infection, or hypoxia. This condition can lead to corneal scarring and vision impairment.

Parent term (use hpo.jax.org/app)
Corneal neovascularization (11496 and 11492)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Herpetic eye disease

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001231
- label: Corneal stromal vascularisation
- parents: [Corneal neovascularization (HP:0011496)](https://hpo.jax.org/app/browse/term/HP:0011496)
- pmid: 
- def: The abnormal growth of blood vessels into the corneal stroma, often associated with ocular herpes, chronic inflammation, infection, or hypoxia. This condition can lead to corneal scarring and vision impairment.
- comment: This feature may be observed in herpetic eye disease.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10874
- synonyms: Stromal vascularization of the cornea(NONE)|Corneal stromal neovascularization(NONE)
